.github/workflows/chart-release-manual.yml

```diff
@@ -0,0 +1,27 @@
+name: Push Charts to helm-charts-registry manually with specified version
+
+on:
+  workflow_dispatch:
+    inputs:
+      chart_version:
+        type: string
+        description: "Chart version"
+        required: true
+
+concurrency:
+  group: schulportal-client-chart-manual
+  cancel-in-progress: true
+
+jobs:
+  scan:
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-helm-kics.yaml@1.3.1
+    permissions:
+      contents: read
+  release:
+    needs: scan
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/chart-release.yaml@1.3.1
+    secrets: inherit
+    with:
+      chart_name: spsh-client
```

Comment:
must not be done in this ticket - but here again the question arises if the helm chart should not be called schulportal-client